docker-compose.yaml

```diff
@@ -0,0 +1,49 @@
+services:
+  api:
+    image: pramsey/pg_featureserv
+    ports:
+      - "0.0.0.0:${OSM_PG_FEATURESERV_API_SYSTEM_PORT}:9000"
+    environment:
+      - DATABASE_URL=postgres://${OSM_PG_FEATURESERV_POSTGRES_USER}:${OSM_PG_FEATURESERV_POSTGRES_PASS}@db/${OSM_PG_FEATURESERV_POSTGRES_DBNAME}
+    depends_on:
+      db:
+        condition: service_healthy
+  db:
+    image: postgis/postgis:latest
+    ports:
+      - "0.0.0.0:${OSM_PG_FEATURESERV_DB_SYSTEM_PORT}:5432"
+    volumes:
+      - osm_pg_featureserv_db_data:/var/lib/postgresql/data
+      - ./data:/data
+    environment:
+      # required to initialize database
+      - POSTGRES_DB=${OSM_PG_FEATURESERV_POSTGRES_DBNAME}
+      - POSTGRES_USER=${OSM_PG_FEATURESERV_POSTGRES_USER}
+      - POSTGRES_PASSWORD=${OSM_PG_FEATURESERV_POSTGRES_PASS}
+      # required to be able to execute stuff in psql
+      - PGDATABASE=${OSM_PG_FEATURESERV_POSTGRES_DBNAME}
+      - PGUSER=${OSM_PG_FEATURESERV_POSTGRES_USER}
+      - PGPASSWORD=${OSM_PG_FEATURESERV_POSTGRES_PASS}
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready",
+              "-d", "${OSM_PG_FEATURESERV_POSTGRES_DBNAME}",
+              "-U", "${OSM_PG_FEATURESERV_POSTGRES_USER}"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+  osm2pgsql:
+    image: iboates/osm2pgsql:latest
```

Comment:
we should use fixed versions to ensure reproducability

Reply:
Done